### PR TITLE
Fabo/add a button to show the address on the Ledger

### PR DESCRIPTION
**Pull request by faboweb**
image

**Description:**

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: src/components/common/AppMenu.vue [hunk 1]
```diff
@@ -214,6 +220,11 @@ export default {
   margin: 2.5rem 1rem 1rem;
 }
 
+.show-on-ledger {
+  display: flex;
```
Comment (jbibla) [inline src/components/common/AppMenu.vue]: why display: flex?

File: src/components/common/AppMenu.vue [hunk 2]
```diff
@@ -4,6 +4,9 @@
       <div>
         <h3>Your Address</h3>
         <Bech32 :address="address || ''" />
+        <a class="show-on-ledger" v-if="session.sessionType === 'ledger'" @click="showAddressOnLedger()">
+          Show On Ledger
```
Comment (jbibla) [inline src/components/common/AppMenu.vue]:
```suggestion
          Show on Ledger
```

File: src/vuex/modules/ledger.js [hunk 3]
```diff
@@ -9,6 +9,27 @@ export default () => {
...
   const actions = {
+    async showAddressOnLedger() {
+      const { default: Ledger } = await import("@lunie/cosmos-ledger")
+
+      const ledger = new Ledger({
+        testModeAllowed: state.externals.config.testModeAllowed
+      })
+
+      try {
+        await ledger.confirmLedgerAddress()
```
Comment (jbibla) [inline src/vuex/modules/ledger.js]: what happens if you click this button but the ledger is not plugged in? or on screensaver mode? or not in Cosmos app?

Comment (faboweb) [reply src/vuex/modules/ledger.js]: 🤷‍♂

Comment (faboweb) [reply src/vuex/modules/ledger.js]: image

**Pull request merged**